src/perl/lib/wtsi_clarity/epp/generic/cherrypick_stamper.pm

```diff
@@ -32,24 +38,101 @@ override 'run' => sub {
   return;
 };
 
+# Transforms an io_node XML::Element into a useful data representation
+sub _augment_input {
```

Comment:
Maybe, this can also go to the stamper_common.
And also the _group_by_container...